pubnub-gson/src/main/java/com/pubnub/internal/v2/subscription/SubscriptionImpl.kt

```diff
@@ -51,4 +69,85 @@ class SubscriptionImpl(
             }
         }
     }
+
+    override fun setOnMessage(onMessageHandler: OnMessageHandler?) {
+        setOnMessageHandler(onMessageHandler)
```

Comment:
why add a layer of indirection, why not put `emitterHelper.onMessage =` here?

Reply:
Good point. I will remove this additional layer.